marketplace-builder/Cargo.toml

```diff
@@ -11,9 +11,9 @@ libp2p = ["sequencer/libp2p"]
 
 [dependencies]
 anyhow = { workspace = true }
-async-broadcast = "0.7.0"
+async-broadcast = { workspace = true }
 async-compatibility-layer = { workspace = true }
-async-lock = "3.4.0"
+async-lock = "2.2"
```

Comment:
Why don't we update async-lock to 3.4 in all deps?

Reply:
We can, but we should start with HotShot, otherwise we run into incompatibilities